lib/static/new-ui/components/Screenshot/index.tsx

```diff
@@ -21,12 +21,25 @@ interface ScreenshotProps {
         size?: ImageSize;
     };
     style?: React.CSSProperties;
+    onMouseEnter?: (e: React.MouseEvent<HTMLImageElement>) => void;
+    onMouseLeave?: (e: React.MouseEvent<HTMLImageElement>) => void;
+    onMouseMove?: (e: React.MouseEvent<HTMLImageElement>) => void;
 }
 
-export function Screenshot(props: ScreenshotProps): ReactNode {
-    const imageRef = useRef<HTMLImageElement | null>(null);
+export const Screenshot = forwardRef<HTMLImageElement, ScreenshotProps>(function Screenshot(props, imageRef): ReactNode {
```

Comment:
The essence of changes in this file: allow passing ref to img element and allow to subscribe to events.